home/index.js

```diff
@@ -11,7 +11,7 @@ const newSlides = () => {
 newSlides();
 
 let textarea = document.querySelector("textarea");
-let messageThank = document.querySelector(".messageThank");
+let messageThank = document.querySelector(".message_thank");
```

Comment:
this is not snake, the snake is message-thank 
all your new CSS names of classes need to change btw

Reply:
Done